<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>main</class>
 <widget class="QMainWindow" name="main">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>811</width>
    <height>579</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Odoo</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="Lable">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>140</y>
      <width>160</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>30</pointsize>
     </font>
    </property>
    <property name="contextMenuPolicy">
     <enum>Qt::NoContextMenu</enum>
    </property>
    <property name="text">
     <string>扫描打印</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>260</y>
      <width>511</width>
      <height>80</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="0" column="0">
      <widget class="QLabel" name="label_11">
       <property name="font">
        <font>
         <family>Arial</family>
         <pointsize>22</pointsize>
        </font>
       </property>
       <property name="text">
        <string>请输入：</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="inputline">
       <property name="font">
        <font>
         <pointsize>20</pointsize>
        </font>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="status_info">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>540</y>
      <width>54</width>
      <height>12</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>811</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuLogin">
    <property name="title">
     <string>用户</string>
    </property>
    <addaction name="actionLogin"/>
   </widget>
   <widget class="QMenu" name="menuERP">
    <property name="title">
     <string>扫描配置</string>
    </property>
    <widget class="QMenu" name="print_method">
     <property name="title">
      <string>打印方法</string>
     </property>
    </widget>
    <widget class="QMenu" name="print_model">
     <property name="title">
      <string>打印模型</string>
     </property>
    </widget>
    <addaction name="print_method"/>
    <addaction name="print_model"/>
    <addaction name="actionselectbartenderpath"/>
   </widget>
   <addaction name="menuLogin"/>
   <addaction name="menuERP"/>
  </widget>
  <widget class="QStatusBar" name="statusBar">
   <property name="enabled">
    <bool>true</bool>
   </property>
  </widget>
  <action name="actionLogin">
   <property name="text">
    <string>登录</string>
   </property>
  </action>
  <action name="actionselectbartenderpath">
   <property name="text">
    <string>打印软件路径</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
